Horion/Module/Modules/AutoTotem.cpp

```diff
@@ -21,29 +22,51 @@ void AutoTotem::onTick(C_GameMode* gm) {
 	if (g_Data.getLocalPlayer() != nullptr) 
 	{
 		C_ItemStack* i = g_Data.getLocalPlayer()->getEquippedTotem();
-
-		if (i->item == NULL && delay > 3) 
+		if (delay > tickDelay) 
 		{
-			C_PlayerInventoryProxy* supplies = g_Data.getLocalPlayer()->getSupplies();
-			C_Inventory* a = supplies->inventory;
-			for (int i = 0; i < 36; i++) {
-				C_ItemStack* test = a->getItemStack(i);
-				if (test->item != NULL) {
-					C_Item* yikes = *test->item;
-					if (yikes->itemId == 450) {
-						g_Data.getLocalPlayer()->consumeTotem();
-						g_Data.getLocalPlayer()->setOffhandSlot(test);
-					}
-
-				}
+			if (isSOH == true)
```

Comment:
add this check here as well: `i->item == NULL`

Reply:
i havent tested with the SOH autototem, but im pretty sure that that check messes stuff up if there is an item clientsidedly but not serversidedly (visual flickering for example) ill add it regardless